easycv/models/backbones/deitiii.py

```diff
@@ -0,0 +1,365 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
```

Comment:
add unittest for the model

Reply:
done

Reply:
What is the difference between DeiTIII and vit_transfomer_dynamic? Can it be merged. https://github.com/alibaba/EasyCV/blob/master/easycv/models/backbones/vit_transfomer_dynamic.py

Reply:
done